### DISCUSSION TITLE
How to close path of lines to make a shape?

### DISCUSSION THREAD

**Discussion by eolach** [zsviczian/obsidian-excalidraw-plugin#619] [Q&A]
Learning to use Excalidraw in Obsidian, mostly on iPad Pro. Hoping more expert people can help on this curve!

When moving a point at one end of a line over the point at the other end of the same line, which keystrokes are required to attach the points to form a shape? Have not succeeded in doing this on iPad or on Mac.

**Comment by YesterKo** [comment]
I don't believe you need any keystroke, just move the lines over them close enough and if you set it to have a background it will automatically appear

https://user-images.githubusercontent.com/12015722/172052819-17741a17-aa9a-4f4f-b6c5-6b61d371fa61.mp4

.
